frontend/src/router.tsx

```diff
@@ -149,6 +151,30 @@ const AppRouter: React.FC = () => {
         claim={Claims.ADMIN_USERS}
         title={getTitle('Edit User')}
       />
+      <AppRoute
+        protected
+        path="/admin/agencies"
+        component={ManageAgencies}
+        layout={AuthLayout}
+        claim={Claims.ADMIN_USERS}
+        title={getTitle('Agency Management')}
+      />
+      <AppRoute
+        protected
+        path="/admin/agency/:id"
```

Comment:
I like that you kept nice SEO URL paths.  However you may want to use the same path conventions we use in the API.  I don't have a preference though.